.version.sh

```diff
@@ -45,6 +45,9 @@ cd -
 
 # Publish the anycloud/node to NPM with the new version
 cd anycloud/node
+# make sure tsc/typescript is installed which is required for the build script
+yarn install
+yarn build
 rm -rf node_modules/ bin
```

Comment:
Maybe also add a comment here that we delete these so they don't accidentally end up in the published tarball since `bin` is populated based on your OS.